python/lsst/ip/isr/isrTask.py

```diff
@@ -768,10 +768,22 @@ class IsrTaskConfig(pipeBase.PipelineTaskConfig,
     )
 
     # Amp offset correction.
+    # TODO: Remove this config field on DM-45526.
     doAmpOffset = pexConfig.Field(
         doc="Calculate and apply amp offset corrections?",
         dtype=bool,
         default=False,
+        deprecated="This field is no longer used. Will be removed after v28."
```

Comment:
I think we want to keep this, and get rid of the two new configs.  `IsrTaskConfig.doAmpOffset` then controls whether to pass the exposure to the `IsrTask.ampOffset` task for the offsets to be measured, and the `IsrTask.ampOffset.doApplyAmpOffset` then controls whether those measured values are subtracted.  This reduces the number of config options, and I think ensures everything is consistent without needing extra validation.